share/cmake/FindLCMS2.cmake

```diff
@@ -0,0 +1,69 @@
+# Locate lcms2
+#
+# This module defines
+#  LCMS2_FOUND, if false, do not try to link to lcms2
+#  LCMS2_LIBRARY, where to find lcms2
+#  LCMS2_INCLUDE_DIR, where to find lcms2.h
+#  LCMS2_VERSION, the version of the library
+#
+# By default, the dynamic libraries of lcms2 will be found. To find the static ones instead,
+# you must set the LCMS2_STATIC_LIBRARY variable to TRUE before calling find_package(LCMS2 ...).
+#
+include(FindPkgConfig)
+
+# attempt to find static library first if this is set
+if(LCMS2_STATIC_LIBRARY)
+	set(LCMS2_STATIC liblcms2.a)
```

Comment:
That's a linux specific name.
For Windows, the names of the 'external' libraries are: lcms2.lib, libyamll-cppmd.lib and tinyxml.lib.